packages/firebase_auth/lib/firebase_auth.dart

```diff
@@ -186,7 +190,7 @@ class FirebaseAuth {
   final Map<int, Map<String, dynamic>> _phoneAuthCallbacks =
       <int, Map<String, dynamic>>{};
 
-  FirebaseApp app;
+  final FirebaseApp app;
```

Comment:
Does this needs to be public?

Reply:
Yea, it's public for the native API, so I think it should be public here as well. https://firebase.google.com/docs/reference/android/com/google/firebase/auth/FirebaseAuth.html#getApp()